haskell.nix/library/.github/workflows/check-outdated.yml

```
@@ -0,0 +1,39 @@
+on:
+  schedule:
+    - cron: '0 10 * * 1'
+
+jobs:
+  check-outdated:
+    runs-on: [self-hosted, nix]
+    outputs:
+      msg: ${{ steps.outdated.outputs.outdated }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: check outdated packages
+        id: outdated
+        run: |
+          function output {
+            echo "outdated<<EOF" >> $GITHUB_OUTPUT
+            echo "Repository https://github.com/${{ github.repository }} has outdated haskell dependencies" >> $GITHUB_OUTPUT
+            echo "$msg" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          }
+
+          nix develop -c cabal update
```

There is no `devShells.default`, AFAICS
```suggestion
          nix develop .#ci -c cabal update
```